.npmignore

```diff
@@ -0,0 +1,173 @@
+### VisualStudioCode template
```

Comment:
following the article you quoted in the PR description, this list should be added to .gitignore not .npmignore. Having said that, I would try to reduce this to something reasonable and not list things we don't use for sure (nyc for example)

Reply:
Yes, I think you're right. I already ran into an issue where some dir / file was mistakenly ignored by such a all-encompassing gitignore / npmignore template. Shall we just remove this npmignore and keep gitignore as is? Or do you see something we should definitely add to gitignore right away?